docs/UserGuide.md

```diff
@@ -102,22 +102,20 @@ To view the full record of a student, use the `view` command instead.
 
 Format: `view all`
 
-### Editing a person : `edit`
+### Editing a student : `edit`
 
-Edits an existing person in the address book.
+Edits the respective details of an existing student in the class list.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Format: `edit [index] n/[name of student] id/[id of student] pn/[name of parent]…​`
 
```

Comment:
It might be better to include all the tags that can be edited or note that all tags can be edited.

Reply:
Thanks for the suggestion @maxtance ! Have edited accordingly to add a mention that all tags can be edited (as of now).
As for the list of tags that is actually a good note, I think we can consider adding this into the above "Notes about Command Format" portion in the future as these tags will be used in multiple commands like adding a new student.